extensions/toggl-track/src/toggl/auth.ts

```diff
@@ -34,9 +34,9 @@ export const authenticatedFetch = (token: string, baseUrl: string) => ({
     }
     return (await response.json()) as T;
   },
-  put: async <T>(endpoint: string, body: unknown): Promise<T> => {
+  patch: async <T>(endpoint: string, body: unknown): Promise<T> => {
     const response = await fetch(baseUrl + endpoint, {
-      method: "PUT",
+      method: "PATCH",
```

Comment:
Did they change the api to do patch requests instead of put requests, or did we change the logic of this extension?

Reply:
The API was changed from `PUT` to `PATCH`